Src/ILGPU/Resources/ErrorMessages.resx

```diff
@@ -124,7 +124,7 @@
     <value>An entry point must receive at least one index parameter</value>
   </data>
   <data name="InvalidEntryPointIndexParameterOfWrongType" xml:space="preserve">
-    <value>An entry point must receive a compatible index type as first parameter</value>
+    <value>An implicitly grouped entry point must receive an int index type as first parameter, long indecies will not work.</value>
```

Comment:
Question: typo "indecies" => "indices"?

Reply:
Totally missed that, I will fix it when I get home in ~ 90 minutes.